plugins/catalog-react/src/utils/filters.ts

```diff
@@ -28,6 +29,24 @@ export function reduceCatalogFilters(
   }, {} as Record<string, string | symbol | (string | symbol)[]>);
 }
 
+export function reduceBackendCatalogFilters(filters: EntityFilter[]) {
+  const backendCatalogFilters: Record<
+    string,
+    string | symbol | (string | symbol)[]
+  > = {};
+
+  filters.forEach(filter => {
+    if (
+      filter instanceof EntityKindFilter ||
+      filter instanceof EntityTypeFilter
```

Comment:
this function is only used by `EntityListProvider` to compute the "server side" filters for retrieving the entities.
The previous implementation was looking for `EntityFilter` implementing `getCatalogFilters`. Since now more filters implement `getCatalogFilters` I had to introduce this check in order to keep retro-compatibility, otherwise the backendEntities in `EntityListProvider` will contain less elements than usual (which might break custom filters).

But you are touching an interesting point. If you think there might be adopters setting custom server side filters an alternative could be to "flip the logic":

```js
if(!(filter instanceof EntityTagFilter || filter instanceof EntityOwnerFilter || filter instanceof EntityLifecycleFilter ...)) {
    Object.assign(backendCatalogFilters, filter.getCatalogFilters());
}
```

so we could hardcode the opposite filters giving to adopters the possibility to extend the logic with new filters. But I'm not sure this is going to be more robust than the current implementation 🤔

Reply:
Yep let's flip it around 👍